admin/widgets.rb

```diff
@@ -0,0 +1,171 @@
+      f.has_many :permissions do |p|
+        p.input :role, include_blank: "Alle"
+        p.input :action, as: :select, collection: Goldencobra::Permission::PossibleActions, include_blank: false
+        p.input :_destroy, as: :boolean
+      end
+    end
+    f.inputs I18n.t('active_admin.widget.article') do
+      f.input :articles, label: I18n.t('active_admin.widget.article_label'), hint: I18n.t('active_admin.widget.article_hint'), as: :select, collection: Goldencobra::Article.order("title ASC"), input_html: { class: 'chosen-select', "data-placeholder" => I18n.t('active_admin.widget.article_placeholder') }
+    end
+    f.actions
+  end
+
+  sidebar :layout_positions, only: [:edit] do
+    ul do
+      Goldencobra::Widget.tag_counts_on(:tags).map(&:name).each do |wtag|
+        li do
+          wtag
+        end
+      end
+    end
+  end
+
+  sidebar :help, only: [:edit, :show] do
+    render "/goldencobra/admin/shared/help"
+  end
+
+  index download_links: proc{ Goldencobra::Setting.for_key("goldencobra.backend.index.download_links") == "true" }.call do
+    selectable_column
+    column I18n.t('active_admin.widget.title'), :title, sortable: :title do |widget|
+      link_to(widget.title, edit_admin_widget_path(widget), title: I18n.t('active_admin.widget.title1'))
+    end
+    column I18n.t('active_admin.widget.position'), :tag_list, sortable: false
+    column I18n.t('active_admin.widget.active'), :active, sortable: :active do |widget|
+      raw("<span class='#{widget.active ? I18n.t('active_admin.widget.online') : I18n.t('active_admin.widget.offline')}'>#{widget.active ? I18n.t('active_admin.widget.online') : I18n.t('active_admin.widget.offline')}</span>")
+    end
+    column I18n.t('active_admin.widget.sorternr'), :sorter
+    column I18n.t('active_admin.widget.standard'), :default, sortable: :default do |widget|
+      widget.default ? "Ja" : "Nein"
+    end
+    column I18n.t('active_admin.widget.css_classes'), :css_name
+    column I18n.t('active_admin.widget.id'), :id_name
+    column I18n.t('active_admin.widget.access') do |widget|
+      Goldencobra::Permission.restricted?(widget) ? raw("<span class='secured'>beschränkt</span>") : ""
+    end
+    column "" do |widget|
+      result = ""
+      result += link_to(t(:edit), edit_admin_widget_path(widget), class: "member_link edit_link edit", title: I18n.t('active_admin.widget.title_widget_edit'))
+      result += link_to(t(:delete), admin_widget_path(widget), method: :DELETE, "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]), class: "member_link delete_link delete", title: I18n.t('active_admin.widget.title_widget_delete'))
+      raw(result)
+    end
+  end
+
+  show title: :title do
+    panel I18n.t('active_admin.widget.widget') do
+      attributes_table_for widget do
+        [:title, :content, :css_name, :active].each do |a|
+          row a
+        end
+      end
+    end
+    panel I18n.t('active_admin.widget.articles') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.